layer/Dockerfile

```diff
@@ -14,25 +14,16 @@ RUN yum update -y \
 # aws cli
 #
 
-COPY requirements.txt ./
-RUN python -m pip install -r requirements.txt -t /opt/awscli
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+RUN unzip awscliv2.zip
+RUN ./aws/install -i /opt/awscli -b /opt/awscli/bin
```

Comment:
Maybe to assist a future reader, add a comment here linking to the aws docs with these installation instructions.